cli/internal/graph/graph.go

```diff
@@ -0,0 +1,64 @@
+// Package graph contains the CompleteGraph struct and some methods around it
```

Comment:
I just moved this struct out because otherwise there was a cyclical import. I would loourrvee to reduce `run.go` and refactor out some of these structs as we go too though :D. I tried not to get too nerd-sniped by this